PontoCom/View/MainView.swift

```diff
@@ -132,6 +133,9 @@ struct MainView: View {
     }
 }
 
-#Preview {
-    MainView()
+struct MainView_Previews: PreviewProvider {
+    static var previews: some View {
+        MainView()
+            .environmentObject(LoginViewModel())
+    }
```

Comment:
Pode deixar na mesma estrutura do #Preview

```suggestion
#Preview {
     MainView().environmentObject(LoginViewModel())
```